common/button/button-en.html

```diff
@@ -0,0 +1,415 @@
+{
+	"title": "Buttons",
+	"language": "en",
+	"description": "Buttons page including working examples to test how various button styles appear and function",
+	"dateModified": "2023-06-09"
+}
+---
+
+<h2>On this page:</h2>
+
+<ul>
+	<li><a href="#purpose">Purpose</a></li>
+	<li><a href="#basic">Basic use</a></li>
+	<li><a href="#enhanced">Enhanced use</a>
+	<ul>
+	<li><a href="#colours/styles">Colours/Styles</a></li>
+	<li><a href="#calltoaction">Call to Action</a></li>
+	<li><a href="#sizing">Sizing</a></li>
+	<li><a href="#fullwidth">Full Width</a></li>
+	<li><a href="#activestate">Active State</a></li>
+	<li><a href="#disabledstate">Disabled State</a></li>
+	<li><a href="#bghorizontal">Button Groups (Horizontal)</a></li>
+	<li><a href="#bgvertical">Button Groups (Vertical)</a></li>
+	<li><a href="#buttontoolbar">Button Toolbar</a></li>
+	<li><a href="#inputgroups">Input Groups</a></li>
+	<li><a href="#datatoggle">Data-toggle Buttons</a></li>
+	</ul></li>
+</ul>
+
+<h2 id="purpose">Purpose</h2>
+<p>Use buttons to create a call to action that entices the user to submit content from a form. Also, use buttons to take users to a webpage that provides additional information.</p>
+
+
+
+<h2 id="basic">Basic use</h2>
+
+<h3>Default</h3>
+<p>Use the button classes on <code>a</code>, <code>button</code>, or <code>input</code> elements to create visual consistency. The browser shows certain elements as a button. However, there is often a lack of visual consistency for each button type. Each browser can have a different button default appearance.</p>
```

Comment:
```suggestion
```

Reply:
Addressed in commit  "Button: Addressed Review Comments"